### DISCUSSION TITLE
Is there a winget package?

### DISCUSSION THREAD

**Discussion by takashikumagai** [xournalpp/xournalpp#3989] [Q&A] [answered]
Hello folks,

Is there a winget package of Xournal++ for the Windows platform?

Regards,
Takashi K.

**Comment by bhennion** [comment, marked as answer]
Not for now. This has been already requested though: see #3913.

**Reply by takashikumagai** [reply]
Got it, thanks. Seems like some progress has already been made. Let me know if there's anything I can help you with
